src/DssSpell.sol

```diff
@@ -21,52 +21,42 @@ import "dss-exec-lib/DssAction.sol";
 
 import { GemAbstract } from "dss-interfaces/ERC/GemAbstract.sol";
 
-interface RwaLiquidationOracleLike {
-    function tell(bytes32 ilk) external;
+interface DenyLike {
+    function deny(address usr) external;
 }
 
 interface ChainlogLike {
     function removeAddress(bytes32) external;
 }
 
-interface RwaInputConduitLike {
-    function mate(address usr) external;
-    function file(bytes32 what, address data) external;
-}
-
-interface DssVestLike {
-    function yank(uint256) external;
-}
-
 interface ProxyLike {
     function exec(address target, bytes calldata args) external payable returns (bytes memory out);
 }
 
-interface FlliperMomLike {
-    function setOwner(address owner_) external;
-}
-
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/64b0d8425a316c40d1fc464c084d800e3b17c1f3/governance/votes/Executive%20vote%20-%20August%2030%2C%202023.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/ef206389a490089bd06e64c574038f07bfbb7569/governance/votes/Executive%20vote%20-%20September%2013%2C%202023.md' -q -O - 2>/dev/null)"
     string public constant override description =
-        "2023-08-30 MakerDAO Executive Spell | Hash: 0xd4f061429b564e422f18f0289a0f956f0229edbeead40b51ad69d52b27f7e591";
-
-    GemAbstract internal immutable MKR                         = GemAbstract(DssExecLib.mkr());
-    address internal immutable MCD_ESM                         = DssExecLib.esm();
-    address internal immutable MCD_VOW                         = DssExecLib.vow();
-    address internal immutable MCD_FLIPPER_MOM                 = DssExecLib.flipperMom();
-    address internal immutable MIP21_LIQUIDATION_ORACLE        = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-    address internal immutable MCD_VEST_DAI                    = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
-    address internal immutable MCD_VEST_MKR_TREASURY           = DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY");
-    address internal immutable RWA015_A_INPUT_CONDUIT_URN_USDC = DssExecLib.getChangelogAddress("RWA015_A_INPUT_CONDUIT_URN");
-    address internal immutable RWA015_A_INPUT_CONDUIT_JAR_USDC = DssExecLib.getChangelogAddress("RWA015_A_INPUT_CONDUIT_JAR");
+        "2023-09-13 MakerDAO Executive Spell | Hash: 0x214ab69eb9c381276e409b9c58c74a6e090f1105992fabc8986d88091121765f";
+
+    GemAbstract internal immutable MKR      = GemAbstract(DssExecLib.mkr());
```

Comment:
Alignment is off here.
Either remove the trailing spaces before `=`:
```suggestion
    GemAbstract internal immutable MKR = GemAbstract(DssExecLib.mkr());
```

Or align it to the block below:
```suggestion
    GemAbstract internal immutable MKR         = GemAbstract(DssExecLib.mkr());
```